packages/manager/apps/cloud/client/app/dbaas/dbaas-metrics/metrics.js

```diff
@@ -6,13 +6,17 @@ angular.module('managerApp').config(($stateProvider) => {
   };
 
   $stateProvider
+    .state('dbaas', {
+      url: '/dbaas',
+      abstract: true,
+      translations: {
+        value: ['../../common', '../../cloud'],
```

Comment:
I think you should define all translations in the module itself. check with team

Reply:
All translations for LDP are in the module itself. Metrics will be done later when we move metrics out :)